core/src/Pos/Core/Util/LogSafe.hs

```diff
@@ -71,268 +55,17 @@ module Pos.Core.Util.LogSafe
-produce the two desired @Buildable@ instances.
-
-You are not supposed to specify this typeclass in constraints, for that
-purpose use 'BuildableSafe'.
-
-Example of usage:
-
-@
-data Password = Password [Char]
-
-instance BuildableSafeGen Password where
-    buildSafeGen sl (Password chars) = bprint (secureListF sl string) chars
-    -- "sl" stands for "security level"
-@
--}
-class BuildableSafeGen a where
-    buildSafeGen :: LogSecurityLevel -> a -> Builder
-
--- | Builds up @instance Buildable a@ and @instance Buildable (SecureLog a)@
--- assuming provided @instance BuildableSafeGen a@.
--- Suitable for simple types.
---
--- Example: @deriveSafeBuildable ''Nyan@
-deriveSafeBuildable :: TH.Name -> TH.Q [TH.Dec]
-deriveSafeBuildable typeName =
-    let typeQ = TH.conT typeName
-    in [d|
-       instance Buildable $typeQ where
-           build = buildSafeGen unsecure
-
-       instance Buildable (SecureLog $typeQ) where
-           build = buildSafeGen secure . getSecureLog
-       |]
-
--- | Same as 'logMesssage', put to public logs only (these logs don't go
--- to terminal). Use it along with 'logMessageS' when want to specify
--- secret and public log alternatives manually.
-logMessageUnsafeP
-    :: (HasLoggerName m, MonadIO m)
-    => Severity
-    -> Text
-    -> m ()
-logMessageUnsafeP severity t =
-    reify selectPublicLogs $ \s ->
-    execSecureLogWrapped s $ do
-        name <- askLoggerName
-        dispatchMessage name severity t
-
--- | Shortcut for 'logMessageUnsafeP' to use according severity.
-logDebugUnsafeP, logInfoUnsafeP, logNoticeUnsafeP, logWarningUnsafeP, logErrorUnsafeP
-    :: (HasLoggerName m, MonadIO m)
-    => Text -> m ()
-logDebugUnsafeP   = logMessageUnsafeP Debug
-logInfoUnsafeP    = logMessageUnsafeP Info
-logNoticeUnsafeP  = logMessageUnsafeP Notice
-logWarningUnsafeP = logMessageUnsafeP Warning
-logErrorUnsafeP   = logMessageUnsafeP Error
-
-
-type SecuredText = LogSecurityLevel -> Text
-
-getSecuredText :: LogSecurityLevel -> SecuredText -> Text
-getSecuredText = (&)
-
--- | Same as 'logMesssageSP', put to public and secret logs securely.
-logMessageSP
-    :: (HasLoggerName m, MonadIO m)
-    => Severity -> SecuredText -> m ()
-logMessageSP severity securedText = do
-    logMessageS severity $ securedText SecretLogLevel
-    logMessageUnsafeP severity $ securedText PublicLogLevel
-
--- | Shortcut for 'logMessage' to use according severity.
-logDebugSP, logInfoSP, logNoticeSP, logWarningSP, logErrorSP
-    :: (HasLoggerName m, MonadIO m)
-    => SecuredText -> m ()
-logDebugSP   = logMessageSP Debug
-logInfoSP    = logMessageSP Info
-logNoticeSP  = logMessageSP Notice
-logWarningSP = logMessageSP Warning
-logErrorSP   = logMessageSP Error
-
-instance BuildableSafe a => Buildable (SecureLog [a]) where
-    build = bprint (buildSafeList secure) . getSecureLog
+-- | additional instances for
 
 instance Buildable (SecureLog Text) where
```

Comment:
it needs access to low-level logging, therefore it was moved to 'util'. these instances remain here to avoid having `katip` as a dependency in other modules than 'util'. also, module 'util' does not have access to these types, defined higher in the module hierarchy.